docs/content/preview/api/ysql/the-sql-language/statements/streaming_start_replication.md

```diff
@@ -0,0 +1,54 @@
+---
+title: START_REPLICATION statement [YSQL]
+headerTitle: START_REPLICATION
+linkTitle: START_REPLICATION
+description: Use the START_REPLICATION statement to start streaming from a replication slot.
+menu:
+  preview:
+    identifier: streaming_start_replication
+    parent: statements
+type: docs
+---
+
+## Synopsis
+
+Use the `START_REPLICATION` command to start streaming changes from a logical replication slot.
+
+## Syntax
+
+{{%ebnf%}}
+  start_replication,
+  log_sequence_number,
+  replication_option
+{{%/ebnf%}}
+
+## Semantics
+
+Instructs the server to start streaming WAL for logical replication. The output plugin associated with the selected slot is used to process the output for streaming.
+
+### *slot_name*
+
+The name of the slot to stream changes from. This parameter is required, and must correspond to an existing logical replication slot created with [`CREATE_REPLICATION_SLOT`](../streaming_create_repl_slot).
+
+### *log_sequence_number*
+
+The log sequence number from where to start the streaming from.
+
+### *start_replication_option_name*
+
+The name of an option passed to the slot's logical decoding plugin.
+
+### *start_replication_option_value*
+
+Optional value, in the form of a string constant, associated with the specified option.
+
+## Example
+
```

Comment:
Add brief summary - "Start replication from Replication Slot *test_slot* using LSN *0/2* and something something" (explain what is going on)

Reply:
Added

Reply:
But what is 'pub'?

Reply:
Do we need to document these options too https://www.postgresql.org/docs/current/protocol-logical-replication.html ?

Reply:
In this example, we probably need a full example? So creating a table, a publication, and then starting replication.
And maybe even show the output of start_replication.

Reply:
@ddorian 

> Do we need to document these options too https://www.postgresql.org/docs/current/protocol-logical-replication.html ?

So these parameters depend on the output plugin chosen. These parameters are specific to `pgoutput`. I've added links to the respective documentation for `pgoutput`, `yboutput` and `wal2json`. If the user uses any other plugin, they have to find the documentation for that plugin.

> In this example, we probably need a full example? So creating a table, a publication, and then starting replication.
And maybe even show the output of start_replication.

Good point. I've added one. We can't show the output of the start_replication as it streams data via copy protocol. The command doesn't output anything on the shell.